### PR TITLE
Link the Naming Convention

### PR DESCRIPTION
**Pull request by coorasse**
And some other smaller improvements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,16 +5,19 @@ We are a [Rails company](https://rubyonrails.org/foundation), so the most value
 will be found in the parts concerning Rails. But anyways you'll also find a lot about the
 inner workings of Renuo.
 
-* [Ruby on Rails – App Setup Guide](./ruby_on_rails/README.md)
+* [Ruby on Rails – Application Setup Guide](./ruby_on_rails/README.md)
```
Comment (coorasse) [inline README.md:8]: I want to keep the extended name "Application" everywhere. It's more consistent.

File: README.md [hunk 2]
```diff
@@ -5,16 +5,19 @@ We are a [Rails company](https://rubyonrails.org/foundation), so the most value
...
 If you are reading this document, it means that you have to setup a new application.
 A new project started and it's now time to set everything up so that **everyone**,
 in your team, **can start working on it**.
 
 This document will try to be as minimalist as possible and provide you with all the steps to set up the application as
-fast as possible. There are things, in Renuo projects, which are mandatory, other that are suggested.
+fast as possible. There are things, in Renuo projects, which are mandatory, other that are suggested. 
+This guide is the result of more than ten years of experience, so this means two things: it's very robust and very opinionated.
+
+**You are always welcome to challenge the guide and improve it with a Pull Request.**
```
Comment (coorasse) [inline README.md:20]: I want to state this again. It's important that people challenge it

File: naming_conventions.md [hunk 3]
```diff
@@ -6,8 +6,10 @@ asked to do the same.
 
 ## Project Name
 
-The project name `[project-name]` consists of `[base-project]-[sub-project]`, and only uses `[a-z0-9]` and dash `-`.
-No underscore `_`.
+* The project name `[project-name]` only uses `[a-z0-9]` and dash `-`. No underscore `_`.
+* A project name is easy to remember and easy to pronounce. In the best case it consists of one word.
+* The project name should be unique and not too long. 
+* It should not contain any version information.
```
Comment (coorasse) [inline naming_conventions.md:13]: some more best practices I think are important. I, for example, don't like that we have to live with gifcoins2, kingschair2, renuo-website-v3 now. I'd like them to simply be `gifcoins`, `kingschair`, and `renuo-website`

Comment (CuddlyBunion341) [reply naming_conventions.md:13]: I agree.

File: ruby_on_rails/README.md [hunk 4]
```diff
@@ -1,10 +1,11 @@
-# Ruby On Rails
+# Ruby On Rails - Application Setup Guide
 
 This setup will cover a pure, monolithic Rails Applications.
-This is the most frequent type of application we have at Renuo and probably also the easiest to setup.
+This is the most frequent type of application we have at [Renuo](https://renuo.ch) and probably also the easiest to setup.
```
Comment (CuddlyBunion341) [inline ruby_on_rails/README.md:4]:
```suggestion
This is the most frequent type of application we have at [Renuo](https://renuo.ch) and is probably also the easiest to setup.
```

**Review by CuddlyBunion341 — APPROVED** [review]
Looks good to me 👍🏻 !

File: README.md [hunk 5]
```diff
@@ -5,16 +5,19 @@ We are a [Rails company](https://rubyonrails.org/foundation), so the most value
 will be found in the parts concerning Rails. But anyways you'll also find a lot about the
 inner workings of Renuo.
 
-* [Ruby on Rails – App Setup Guide](./ruby_on_rails/README.md)
+* [Ruby on Rails – Application Setup Guide](./ruby_on_rails/README.md)
 
-## Some Notes on the Side
+## Some Notes on the side
```
Comment (rnestler) [inline README.md:10]: We seem to use [Title Case](https://en.wikipedia.org/wiki/Title_case) consistently so I'd keep the noun capitalized.
```suggestion
## Some Notes on the Side
```

File: README.md [hunk 6]
```diff
@@ -5,16 +5,19 @@ We are a [Rails company](https://rubyonrails.org/foundation), so the most value
...
 
 If you are reading this document, it means that you have to setup a new application.
 A new project started and it's now time to set everything up so that **everyone**,
 in your team, **can start working on it**.
 
 This document will try to be as minimalist as possible and provide you with all the steps to set up the application as
 fast as possible. There are things, in Renuo projects, which are mandatory, other that are suggested.
+This guide is the result of more than ten years of experience, so this means two things: it's very robust and very opinionated.
```
Comment (rnestler) [inline README.md:18]: And also possibly outdated?

**Pull request merged**